install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -73,8 +74,10 @@ spec:
               endpoint:
                 properties:
                   external:
+                    description: The external endpoint address that applications can use to connect to the Broker
                     type: string
                   internal:
+                    description: The internal endpoint address that applications can use to connect to the Broker
```

Comment:
Should be a locally addressable IP address or DNS hostname (e.g.`broker.busybox.svc.cluster-domain.example`).

Reply:
Done.